Makefile

```
@@ -21,6 +21,32 @@ include .bingo/Variables.mk
+ifeq ($(shell go env GOOS),windows)
+GITLEAKS_BIN_EXT := .exe
+else
+GITLEAKS_BIN_EXT :=
+endif
+
+GITLEAKS := $(LOCALBIN)/gitleaks$(GITLEAKS_BIN_EXT)
+
+$(LOCALBIN):
+	mkdir -p "$(LOCALBIN)"
+
+$(GITLEAKS): $(LOCALBIN)/.gitleaks-$(GITLEAKS_VERSION).stamp
```

checkmake (0.3.2)

[warning] Target "$(GITLEAKS)" should be declared PHONY.

(phonydeclared)